Dfe.PrepareConversions/Dfe.PrepareConversions/Pages/TaskList/Decision/RecordADecision.cshtml.cs

```diff
@@ -20,7 +20,7 @@ public async override Task<IActionResult> OnGetAsync(int id)
          ReturnPage = @Links.ProjectList.Index.Page;
          var returnToFormAMatMenu = TempData["returnToFormAMatMenu"] as bool?;
 
-         if (Project.IsFormAMat && returnToFormAMatMenu.HasValue && returnToFormAMatMenu.Value)
+         if (Project.IsFormAMat.HasValue && Project.IsFormAMat.Value && returnToFormAMatMenu.HasValue && returnToFormAMatMenu.Value)
```

Comment:
`    public static bool IsFormAMat(ProjectType project)
    {
        // Check if 'IsFormAMat' is not null and its value is true.
        return project.IsFormAMat.HasValue && project.IsFormAMat.Value;
    }`
    
   This call happens lots in this PR, is it worth extracting to the above to make changes in the future easier? Or if there is unlikely to be changes in the future ignore this